packages/analytics/dashboard-renderer/src/components/QueryDataProvider.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <!-- Loading state -->
+  <KSkeleton
+    v-if="isLoading || (!data && !hasError)"
+    class="chart-skeleton"
+    type="table"
+  />
+  <!-- Error state -->
```

Comment:
nit: can we remove these HTML comments so they don't render in the DOM?

Reply:
Wow, that's really frustrating that they render in the DOM.  Done.